lokinet/lokinet-base.dockerfile

```diff
@@ -1,24 +1,34 @@
-FROM debian:stable AS lokinet-base
+#use argument instead of lsb-release
+ARG DEBIAN_RELEASE=bullseye 
+
+FROM debian:${DEBIAN_RELEASE}-slim AS lokinet-base
 ENV container docker
+
+ENV RELEASE=${DEBIAN_RELEASE:-bullseye}
+#Add oxen public key 
+ADD --chmod=644 --chown=_apt https://deb.oxen.io/pub.gpg /etc/apt/trusted.gpg.d/lokinet.gpg
+
 # set up packages
-RUN /bin/bash -c 'echo "man-db man-db/auto-update boolean false" | debconf-set-selections'
-RUN /bin/bash -c 'apt-get -o=Dpkg::Use-Pty=0 -q update && apt-get -o=Dpkg::Use-Pty=0 -q dist-upgrade -y && apt-get -o=Dpkg::Use-Pty=0 -q install -y --no-install-recommends ca-certificates curl iptables dnsutils lsb-release systemd systemd-sysv cron conntrack iproute2 python3-pip wget'
-RUN /bin/bash -c 'curl -so /etc/apt/trusted.gpg.d/lokinet.gpg https://deb.oxen.io/pub.gpg'
-RUN /bin/bash -c 'echo "deb https://deb.oxen.io $(lsb_release -sc) main" > /etc/apt/sources.list.d/lokinet.list'
-RUN /bin/bash -c 'apt-get -o=Dpkg::Use-Pty=0 -q update && apt-get -o=Dpkg::Use-Pty=0 -q dist-upgrade -y && apt-get -o=Dpkg::Use-Pty=0 -q install -y --no-install-recommends lokinet'
+# not sure if wget, lsb-release and curl are needed (maybe they can be removed to make a smaller image?)
+RUN DEBIAN_FRONTEND=noninteractive \
+    && echo "deb https://deb.oxen.io ${RELEASE} main" > /etc/apt/sources.list.d/lokinet.list \     
+    && echo "man-db man-db/auto-update boolean false" | debconf-set-selections \
+    && apt update -y \
```

Comment:
These should remain the `apt-get` versions rather than `apt`: the latter is meant to be an interactive tool, while `apt-get` is specifically targetted for use in scripts.

Reply:
I don't have an issue with apt-get... but if you parse `DEBIAN_FRONTEND=noninteractive` with the correct arguments you would only see some random warnings that can be ignored. (I'm not expert with debian though) I wanted to get it running in an alpine container but building from source caused some challenges that I may deal with later.

Reply:
which image are you using that does not have `apt-get`?

Reply:
The image I use does have apt-get. There's not much difference between the two (apt & apt-get) IMHO. I just wanted to make things cleaner without the extra arguments to apt-get.

Reply:
`apt-get` is the correct tool here as it is used for scripts and automation like docker, `apt` is not.

Reply:
Fair enough, it can be reverted if you like. I honestly don't see the difference but if it's best practice - I have no objection.

Reply:
i do like the use of multi line formatting here, easier to read. but yes, please use `apt-get` here. thanks.